app/views/proposals/detail/_request_details.html.haml

```diff
@@ -0,0 +1,85 @@
+.row
+  .medium-12.column
+    .card#request-details-card
+      - if @client_data_instance && @proposal.client_slug == "ncr"
```

Comment:
piling on: if we are hardcoding client-specific values in the templates, we are doing it wrong.